channels/CMakeLists.txt

```diff
@@ -176,6 +176,13 @@ macro(add_channel_client_library _module_prefix _module_name _channel_name _dyna
   add_library(${_module_name} OBJECT ${${_module_prefix}_SRCS})
   set_property(TARGET ${_module_name} PROPERTY FOLDER "Channels/${CHANNEL_NAME}/Client")
 
+  if(UNIX AND WITH_SSO_MIB)
```

Comment:
this should only be required at one place, `client/common` as the lib is not used anywhere else

Reply:
you can just do a forward declaration of the type and hide details

Reply:
Cleaned up by moving into client.h but still unsure about the right handling of the type. Right now, I do the following, similar to two other flags that are already there:
```c
#if defined(WITH_SSO_MIB)
		ALIGN64 MIBClientApp* mibClientApp; /**< (offset 10) */
#else
		UINT64 reserved4;
#endif
```